test/e2e/multihoming.go

```diff
@@ -636,6 +642,48 @@ var _ = Describe("Multi Homing", func() {
 			),
 		)
 
+		Context("localnet OVN-K secondary network", func() {
+			const (
+				clientPodName     = "client-pod"
+				nodeHostnameKey   = "kubernetes.io/hostname"
+				servicePort       = 9000
+				dockerNetworkName = "underlay"
+			)
+
+			table.DescribeTable(
+				"with a service running on the underlay",
+				func(netConfigParams networkAttachmentConfigParams, clientPodConfig podConfiguration) {
+					netConfig := newNetworkAttachmentConfig(netConfigParams)
+					netConfig.topology = "localnet"
+					netConfig.cidr = secondaryLocalnetNetworkCIDR
+					netConfig.excludeCIDRs = []string{secondaryLocalnetExcludeCIDR}
+
+					netConfig.namespace = f.Namespace.Name
+					clientPodConfig.namespace = f.Namespace.Name
+
+					defer setupLocalnetUnderlay(cs, netConfig)()
```

Comment:
I would prefer to see explicit test setup / teardown in the `BeforeEach` /`AfterEach` blocks.